src/browser/documentation/sidebar-guides/cypher.tsx

```diff
@@ -22,73 +22,104 @@ import React from 'react'
-  <BuiltInGuideSidebarSlide key="second">
+  <BuiltInGuideSidebarSlide key="s2">
     <h3>CREATE</h3>
-    <p className="lead">Create a node</p>
+    <p className="lead">
+      <em>Create a node</em>
+    </p>
     <p>Let's use Cypher to generate a small social graph.</p>
-    <pre className="pre-scrollable code runnable">
-      {'CREATE (ee:Person { name: "Emil", from: "Sweden", klout: 99 })'}
-    </pre>
-    <ul>
-      <li>
-        <code>CREATE</code> clause to create data
-      </li>
-      <li>
-        <code>()</code> parenthesis to indicate a node
-      </li>
-      <li>
-        <code>ee:Person</code> a variable 'ee' and label 'Person' for the new
-        node
-      </li>
-      <li>
-        <code>{'{}'}</code> brackets to add properties to the node
-      </li>
-    </ul>
+    <p>
+      <b>NOTE:</b> This guide assumes that you use an empty graph.{' '}
+    </p>
+    <ol>
+      <li>
+        Click this code block and bring it into the Editor:
+        <pre className="pre-scrollable code runnable">
+          {
+            'CREATE (ee:Person { name: "Emil", from: "Sweden", kloutScore: 99 })'
+          }
+        </pre>
+        <ul>
+          <li>
+            <code>CREATE</code> creates the node.
+          </li>
+          <li>
+            <code>()</code> indicates the node.
+          </li>
+          <li>
+            <code>ee:Person</code> – <code>ee</code> is the node variable and{' '}
+            <code>Person</code> is the node label.
+          </li>
+          <li>
+            <code>{'{}'}</code> contains the properties that describe the node.
+          </li>
+        </ul>
+      </li>
+      <li>
+        Run the Cypher code by clicking the <b>Run</b> button.
+      </li>
+    </ol>
   </BuiltInGuideSidebarSlide>,
-  <BuiltInGuideSidebarSlide key="third">
+  <BuiltInGuideSidebarSlide key="s3">
     <h3>MATCH</h3>
-    <p className="lead">Finding nodes</p>
-    <p>Now find the node representing Emil:</p>
-    <pre className="pre-scrollable code runnable">
-      MATCH (ee:Person) WHERE ee.name = "Emil" RETURN ee;
-    </pre>
-    <ul>
-      <li>
-        <code>MATCH</code> clause to specify a pattern of nodes and
-        relationships
-      </li>
-      <li>
-        <code>(ee:Person)</code> a single node pattern with label 'Person' which
-        will assign matches to the variable 'ee'
-      </li>
-      <li>
-        <code>WHERE</code> clause to constrain the results
-      </li>
-      <li>
-        <code>ee.name = "Emil"</code> compares name property to the value "Emil"
-      </li>
-      <li>
-        <code>RETURN</code> clause used to request particular results
-      </li>
-    </ul>
+    <p className="lead">
+      <em>Find nodes</em>
+    </p>
+    <p>Now, find the node representing Emil.</p>
+    <ol>
+      <li>
+        Click this code block and bring it into the Editor:
+        <pre className="pre-scrollable code runnable">
+          MATCH (ee:Person) WHERE ee.name = "Emil" RETURN ee;
```

Comment:
MATCH (ee:Person) WHERE ee.name = 'Emil' RETURN ee;

Lets try to follow the Cypher recommended string quotations `'this is the recommended style'` not `"this style".`